.bazelrc

```diff
@@ -5,31 +5,29 @@
 # https://bazel.build/reference/command-line-reference#flag--enable_platform_specific_config
 common --enable_platform_specific_config
 
-# https://bazel.build/docs/windows#symlink
-startup --windows_enable_symlinks
-build:windows --enable_runfiles
-
 # Speed up builds by skipping the creation of runfile links
-build --nobuild_runfile_links
+build:linux --nobuild_runfile_links
 
 # https://github.com/bazelbuild/bazel/issues/8195
 build --incompatible_disallow_empty_glob=true
 
+# Allow isolated module extensions for tests.
+common --experimental_isolated_extension_usages
+
 # https://github.com/bazelbuild/bazel/issues/12821
-build --nolegacy_external_runfiles
+build:linux --nolegacy_external_runfiles
```

Comment:
Why linux only?

Reply:
Setting this flag enables runfiles.  We don't want runfiles on Windows